data/korath/korath.txt

```diff
@@ -351,6 +396,60 @@ fleet "Korath Defense"
 		"'olofez" 4
 		"'olofez (Expeller)" 2
 
+fleet "Korath Humanika Tribute Response"
```

Comment:
```suggestion
fleet "Korath Tribute Response (Humanika Efreti)"
```

Reply:
The Efreti have nothing to do with this story.

Reply:
I see this was applied manually already. I've changed it to Humanika Rafeki. Other than the incorrect name, I agree the change is a good one.